### ISSUE TITLE
Some confusion in CrossEntropyLoss

### ISSUE THREAD

**Issue by thangvubk** [open-mmlab/mmdetection#1138]
As the title, there is some confusion in [CrossEntropyLoss](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/losses/cross_entropy_loss.py). For example, the ``weight`` used in ``forward()`` ([L#95](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L89)) is the ``label`` for ``mask_cross_entropy`` ([#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54)). The variable names seems to not reflect their roles. Some suggestions for improvement could be:
1. [#L87](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L87): change ``cls_score`` to ``pred``.
2. [#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54): change ``target`` to ``label``.
3. [#L54](https://github.com/open-mmlab/mmdetection/blob/5c2e68bb4c20573e8144d92b989454efe886fbbb/mmdet/models/losses/cross_entropy_loss.py#L54): change ``label`` to ``mask``, add argument ``weight`` for future use. It would be make ``mask_cross_entropy`` consistent with other CE losses

**Comment by hellock** [comment]
Good point! You can open a PR to improve that.
